src/explore-education-statistics-admin/src/services/footnoteService.ts

```diff
@@ -106,6 +106,11 @@ const footnoteService = {
   deleteFootnote(releaseId: string, id: string): Promise<void> {
     return client.delete(`/releases/${releaseId}/footnotes/${id}`);
   },
+  updateFootnotesOrder(releaseId: string, ids: string[]): Promise<string[]> {
+    return client.patch(`/releases/${releaseId}/footnotes/`, {
+      FootnoteIds: ids,
```

Comment:
For consistency with how we normally call endpoints, this should be `footnoteIds`.